core-bundle/src/Entity/RememberMe.php

```diff
@@ -32,10 +32,10 @@ class RememberMe
     #[GeneratedValue]
     protected int $id;
 
-    #[Column(type: 'binary_string', length: 88, nullable: false, options: ['fixed' => true])]
+    #[Column(length: 88, nullable: false)]
     protected string $series;
 
-    #[Column(type: 'binary_string', length: 64, nullable: false, options: ['fixed' => true])]
+    #[Column(length: 64, nullable: false)]
```

Comment:
```suggestion
    #[Column(length: 88, nullable: false)]
```

<img width="584" alt="" src="https://github.com/contao/contao/assets/1192057/0e498f65-62df-4f2a-ba89-5cf4df1e5e09">

Reply:
see https://github.com/contao/contao/pull/6815